sway-lib-std/src/lib.sw

```diff
@@ -18,5 +18,6 @@ dep block;
 dep token;
 dep ecr;
 dep reentrancy;
+dep vm/lib;
```

Comment:
Do we need a `lib` in the `vm` module?
I'm not saying it's wrong, but we should probably discuss the stdlib's desired structure as it grows.
cc @sezna

Reply:
Yeah it's very possible this is just wrong! My logic was that this way when we add more stuff to the vm or evm sublibrairies, there's no external dependency to update with the new module, we just add it to lib.sw inside the sublibrary itself. Trying to avoid anything to do with the evm leaking 😂

Reply:
Well, `lib.sw` is typically the entry point to a _project_, not a submodule. That being said, we don't really have a precedent yet for what to name a submodule's version of what Rust calls a `mod.rs`. We could do a quick bikeshed -- to me, it is a bit confusing to have two `lib.sw` files

Reply:
How about... `mod.sw`?